lib/scala/runtime-version-manager-test/src/test/scala/org/enso/runtimeversionmanager/components/RuntimeVersionManagerSpec.scala

```diff
@@ -42,19 +43,44 @@ class RuntimeVersionManagerSpec
       )
 
       val runtime = componentsManager.findGraalRuntime(engine)
-      runtime shouldBe empty
+      runtime.value.version shouldEqual GraalVMVersion("23.2.0", "21.0.0")
+      assert(
+        runtime.value.path.startsWith(distributionManager.paths.runtimes),
+        "Engine should be installed in the engines directory."
+      )
     }
 
-    "list installed engines" in {
+    "list installed engines and runtimes" in {
       val componentsManager = makeRuntimeVersionManager()
       val engineVersions =
-        Set(SemVer(0, 0, 0), SemVer(0, 0, 1), SemVer(0, 0, 1, Some("pre")))
+        Set(SemVer(0, 0, 0), SemVer(0, 0, 1), SemVer(0, 1, 0))
+      val runtimeVersions =
+        Set(
+          components.GraalVMVersion("1.0.0", "11"),
```

Comment:
Because it is specified in [fake-releases/enso-0.0.0/manifest.yaml](https://github.com/enso-org/enso/blob/5a7ad6bfe4fe6c0a945e1afb7c3ec3e58b937293/lib/scala/runtime-version-manager-test/src/main/resources/org/enso/runtimeversionmanager/test/fake-releases/enso/enso-0.0.0/manifest.yaml). Note that runtime-version-manager tests uses a fake release provider that fetches the data from the [fake-releases](https://github.com/enso-org/enso/blob/226ccb742043179b8b5620d35bcfb5bd0edfc6d8/lib/scala/runtime-version-manager-test/src/main/resources/org/enso/runtimeversionmanager/test/fake-releases) directory in resources